server/channels/app/remote_cluster.go

```diff
@@ -62,6 +62,12 @@ func (a *App) RegisterPluginForSharedChannels(opts model.RegisterPluginOpts) (re
 		mlog.String("remote_id", rcSaved.RemoteId),
 	)
 
+	// ping the plugin remote immediately if the service is running
+	rcService, _ := a.GetRemoteClusterService()
```

Comment:
@isacikgoz  Unfortunately the service not being available is fully expected since plugins get started before the services.  This ping is really just for cases where someone enables a plugin while the server is running.  I have added a comment to that effect.

Reply:
Is there a reason the plugins are instantiated before the services?

Reply:
Thank you @wiggin77 that makes sense.